docs/setup-nginx-proxy-with-minio.md

```diff
@@ -34,43 +34,24 @@ Note:
 * Replace ``http://localhost:9000``  with your own server name.
 * Add ``client_max_body_size 1000m;`` in the ``http`` context in order to be able to upload large files — simply adjust the value accordingly. The default value is `1m` which is far too low for most scenarios.
 
-### Non Root Configuration
-When a non root configuration is needed adjust the location definition as follows:
+### Proxy requests based on the bucket
+If you want to serve web-application and Minio from the same nginx port then you can proxy the Minio requests based on the bucket name
 
 ```sh
- location ~^/files {
+ # Proxy requests to the bucket "photos" to Minio server running on port 9000
+ location /photos/ {
    proxy_buffering off;
    proxy_set_header Host $http_host;
    proxy_pass http://localhost:9000;
  }
-```
-
-Note:
-
-* Replace `http://localhost:9000` with your own server name.
-* Replace `files` with the desired path. This cannot be `~^/minio` since `minio` is a reserved word in minio. 
```

Comment:
Can we keep this point from the note

Reply:
It can be `/minio` too @nitisht for web-UI access

Reply:
okay, that makes sense